src/core/credentials/transport/ssl/ssl_security_connector.h

```diff
@@ -25,12 +25,14 @@
 #include <grpc/support/port_platform.h>
 #include <stddef.h>
 
+#include <vector>
+
 #include "src/core/credentials/transport/security_connector.h"
 #include "src/core/tsi/ssl_transport_security.h"
 #include "src/core/util/ref_counted_ptr.h"
 
 struct grpc_ssl_config {
-  tsi_ssl_pem_key_cert_pair* pem_key_cert_pair;
+  tsi_ssl_pem_key_cert_pair pem_key_cert_pair;
```

Comment:
I know we moved the list of them to vectors - is it okay to remove the ptr here as well?

Reply:
Ah understood - then this storing ptr is probably for efficiency/ownership. Do you think we should keep it?